### PR TITLE
IGAPP-411: Fix dashboard navigation items cut on small screens

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

**Comment by klinzo** [comment]
This PR does fix this issue, right? https://issues.tuerantuer.org/browse/IGAPP-367

**Comment by SteffiStuffel** [comment]
> This PR does fix this issue, right? https://issues.tuerantuer.org/browse/IGAPP-367

Yeah, seems to be the same issue as 411, if i get it right.

**Comment by klinzo** [comment]
> > This PR does fix this issue, right? https://issues.tuerantuer.org/browse/IGAPP-367
> 
> Yeah, seems to be the same issue as 411, if i get it right.

Nice, then I'll reject it.

File: web/src/modules/layout/components/Header.js [hunk 1]
```diff
@@ -92,7 +92,6 @@ const NavigationBar = styled.div`
   flex: 1 1 0%; /* The % unit is necessary for IE11 */
   align-items: stretch;
   justify-content: center;
-  overflow-y: hidden;
```
Comment (klinzo) [inline web/src/modules/layout/components/Header.js:95]: AFAICS (according to git blame) this line was added to hide the scrollbar. Do you think there is another solution to this problem?

Comment (maxammann) [reply web/src/modules/layout/components/Header.js:95]: I found this snipped to hide scrollbars but keep functionality.

maybe that works better for some weird css reason:

```
 /* Hide scrollbar for Chrome, Safari and Opera */
.example::-webkit-scrollbar {
  display: none;
}

/* Hide scrollbar for IE, Edge and Firefox */
.example {
  -ms-overflow-style: none;  /* IE and Edge */
  scrollbar-width: none;  /* Firefox */
} 
```

Comment (SteffiStuffel) [reply web/src/modules/layout/components/Header.js:95]: I added the code Max suggested, but i can't see any difference. Tested on Safari on iOS, local Chrome-Browser and in the Android Emulator.
Which solution do you prefer / should we merge or rather do you see any difference?

Comment (Schedulaar) [reply web/src/modules/layout/components/Header.js:95]: If I remember correctly, some scroll bar was shown on android... Not sure though

Comment (SteffiStuffel) [reply web/src/modules/layout/components/Header.js:95]: Thanks for the hint with firefox. Tested this on ios and android emulator, but there is also not scrollbar.

ios firefox
android firefox

Comment (Taggotty) [reply web/src/modules/layout/components/Header.js:95]: Could also not reproduce any scrollbar. @klinzo how old was the change with the scrollbar? Maybe we fixed it sometime in between.

Comment (klinzo) [reply web/src/modules/layout/components/Header.js:95]: Okay, nice :+1: I just used firefox and set dimension to an iPhone :) But I guess this is fine then.

**Review by klinzo — APPROVED** [review]
You can write a release note if you want :)

File: release-notes/unreleased/IGAPP-411-fix-menu-bar-on-small-screens.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+issue_key: IGAPP-363
+show_in_stores: true
+platforms:
+  - android
+  - ios
+en: Hide redundant navigation items.
+de: Redundante Navigationselemente werden nun nicht mehr angezeigt.
```
Comment (klinzo) [inline release-notes/unreleased/IGAPP-411-fix-menu-bar-on-small-screens.yml:7]: Nothing against copy and paste but I guess you should change this :D

**Review by klinzo — CHANGES_REQUESTED** [review]

File: release-notes/unreleased/IGAPP-411-fix-menu-bar-on-small-screens.yml (hunk 2 above)
Comment (SteffiStuffel) [reply release-notes/unreleased/IGAPP-411-fix-menu-bar-on-small-screens.yml]: How does this happen?? Sorry. I fixed it.

**Review by klinzo — APPROVED** [review]

**Review by maxammann — APPROVED** [review]

**Pull request merged**